docs/starters.yml

```diff
@@ -1647,3 +1647,15 @@
     - Comes with React Helmet for adding site meta tags
     - Add Disqus
     - Nice Pagination
+- url: https://master.d2f5ek3dnwfe9v.amplifyapp.com/
+  repo: https://github.com/dabit3/gatsby-auth-starter-aws-amplify
+  description: This Gatsby starter uses AWS Amplify to implement authentication flow for signing up signing in users as well as protected client side routing.
+  tags:
+    - aws
+    - authentication
+    - amplify
```

Comment:
```suggestion
```

Reply:
I aligned this base on @DSchau's recommendation of one category for aws amplify

Reply:
We are probably ok with AWS amplify only but I also like the `Authentication` tag. So lets wait for @DSchau to do the final call :)

Reply:
@DSchau? Thoughts?

Reply:
So i talked to @DSchau offline, lets go for this

```
tags:
    - AWS
    - Authentication
```